main.py

```diff
@@ -50,8 +50,8 @@ def run(path='misc/experiments/ssdnet/params.json', resume=False, eval_only=Fals
         start_epoch = checkpoint['epoch']
         print('Model loaded successfully')
 
-        for pg in optimizer.param_groups:
```

Comment:
might as well delete this

Reply:
that line is deleted. What should I delete?